.gitignore

```
@@ -3,3 +3,18 @@ bower_components
 .sass-cache
 unused-because-wordpress.js*
 .wp-svn
+.DS_Store
+*.DS_Store
+*/.DS_Store
+*/*/.DS_Store
+*/*/*/.DS_Store
+*/*/*/*/.DS_Store
+*/*/*/*/*/.DS_Store
+*/*/*/*/*/*/.DS_Store
+*/*/*/*/*/*/*/.DS_Store
+*/*/*/*/*/*/*/*/.DS_Store
+*/*/*/*/*/*/*/*/*/.DS_Store
+*/*/*/*/*/*/*/*/*/*/.DS_Store
```

```suggestion
.DS_Store
```
I think only this does the trick